src/controllers/auth.js

```diff
@@ -80,7 +80,7 @@ module.exports = {
         acceptTerms,
         address,
       });
-      delete newUser.password_hash;
```

Comment:
This method was not working properly so I changed it to `newUser.password_hash = undefined;`